base/src/main/java/io/spine/validate/MessageValidator.java

```diff
@@ -149,22 +156,22 @@ public void visitDistinct(DistinctConstraint constraint) {
 
     @Override
     public void visitGoesWith(GoesConstraint constraint) {
-        FieldValue value = message.valueOf(constraint.field());
+        FieldDeclaration field = constraint.field();
+        FieldValue value = message.valueOf(field);
         Optional<FieldDeclaration> declaration = withField(message, constraint);
-        checkState(declaration.isPresent(),
-                   "Field `%s` noted in `(goes).with` option is not found.",
-                   constraint.optionValue()
-                             .getWith());
+        String withFieldName = constraint.optionValue().getWith();
+        checkState(
+                declaration.isPresent(),
+                "Field `%s` noted in `(goes).with` option is not found.",
```

Comment:
Please replace “noted” with specified. And please start with “The” before “field”.